src/vm/functions/mod.rs

```diff
@@ -232,6 +233,7 @@ pub fn lookup_reserved_functions(name: &str) -> Option<CallableType> {
             DeleteEntry => SpecialFunction("special_delete-entry", &database::special_delete_entry),
             TupleCons => SpecialFunction("special_tuple", &tuples::tuple_cons),
             TupleGet => SpecialFunction("special_get-tuple", &tuples::tuple_get),
+            TupleSet => SpecialFunction("special_set-tuple", &tuples::tuple_set),
```

Comment:
Both arguments are evaluated, so this should be a regular native function rather than a special.

Reply:
Yep -- this can be a regular native function.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2117/commits/6f976eb11225e8f4eb4bb4ed4ba96b7c20c3983f